narps_open/pipelines/matlabbatch_UK24.m

```diff
@@ -0,0 +1,292 @@
+% 9 nuisance : 6 head motion parameter regressors, 2 tissue signal regressors, and 1 scrubbing regressor
+
+matlabbatch{end+1}.spm.stats.fmri_spec.dir = '<UNDEFINED>';
+matlabbatch{end}.spm.stats.fmri_spec.timing.units = 'secs';
+matlabbatch{end}.spm.stats.fmri_spec.timing.RT = '<UNDEFINED>';
+matlabbatch{end}.spm.stats.fmri_spec.timing.fmri_t = 16;
+matlabbatch{end}.spm.stats.fmri_spec.timing.fmri_t0 = 8;
+matlabbatch{end}.spm.stats.fmri_spec.sess.scans = '<UNDEFINED>';
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(1).name = 'gain';
+% Below we should include the 'onset' values from sub-001_task-MGT_run-01_events
+% for which we have participant_response=accept (either weakly or strongly)
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(1).onset = '<UNDEFINED>';
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(1).duration = 4;
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(1).pmod(1).name = 'gain_value';
+% Below we should include the 'gain' values from sub-001_task-MGT_run-01_events
+% for which we have participant_response=accept (either weakly or strongly)
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(1).pmod(1).param = '<UNDEFINED>';
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(1).pmod(1).poly = 1;
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(1).pmod(2).name = 'gain_RT';
+% Below we should include the 'RT' values from sub-001_task-MGT_run-01_events
+% for which we have participant_response=accept (either weakly or strongly)
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(1).pmod(2).param = '<UNDEFINED>';
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(1).pmod(2).poly = 1;
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(1).orth = 1;
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(2).name = 'loss';
+% Below we should include the 'onset' values from sub-001_task-MGT_run-01_events
+% for which we have participant_response=accept (either weakly or strongly)
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(2).onset = '<UNDEFINED>';
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(2).duration = 2;
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(2).tmod = 0;
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(2).pmod(1).name = 'loss_value';
+% Below we should include the 'loss' values from sub-001_task-MGT_run-01_events
+% for which we have participant_response=accept (either weakly or strongly)
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(2).pmod(1).param = '<UNDEFINED>';
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(2).pmod(1).poly = 1;
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(2).pmod(2).name = 'loss_RT';
+% Below we should include the 'RT' values from sub-001_task-MGT_run-01_events
+% for which we have participant_response=accept (either weakly or strongly)
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(2).pmod(2).param = '<UNDEFINED>';
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(2).pmod(2).poly = 1;
+matlabbatch{end}.spm.stats.fmri_spec.sess.cond(2).orth = 1;
```

Comment:
I could not find the corresponding parameter in nipype

Reply:
https://github.com/nipy/nipype/issues/2811

Reply:
Thanks for finding this thread @bclenet! As far as I can tell here we are trying to keep the orthogonolisation which seems to be the default (it looks like it's the switching off that is not implemented yet). If that's true then we should be able to proceed.